front_end/content.py

```diff
@@ -154,6 +167,12 @@ def create_sqlite_tables(self):
         else:
             print("Error! Cannot create a database connection.")
 
+    def add_column(self):
```

Comment:
The name of this function suggests that it is generic to any column, but it appears to be adding one specific column. I think it should either be more generic or else the name should be more specific. Or maybe better, shouldn't this be in a migration script only rather than here?

Reply:
Sorry, this function was just a temporary method so I could make changes to the database one by one. It isn't needed at all.